test/perl/TimescaleNode.pm

```diff
@@ -6,8 +6,11 @@
 # routines for setup.
 
 package TimescaleNode;
-use parent qw(PostgresNode);
-use TestLib qw(slurp_file);
+use if $ENV{PG_VERSION_MAJOR} >= 15, 'parent', qw(PostgreSQL::Test::Cluster);
+use if $ENV{PG_VERSION_MAJOR} < 15, 'parent', qw(PostgresNode);
+use
+  if $ENV{PG_VERSION_MAJOR} >= 15, 'PostgreSQL::Test::Utils', qw(slurp_file);
+use if $ENV{PG_VERSION_MAJOR} < 15, 'TestLib', qw(slurp_file);
```

Comment:
You can import the `ok` and `is` functions from the right modules here to avoid the duplication below.

```suggestion
use if $ENV{PG_VERSION_MAJOR} < 15, 'TestLib', qw(slurp_file ok is);
use if $ENV{PG_VERSION_MAJOR} >= 15, 'PostgreSQL::Test::Cluster', qw(ok is);
```